CHANGELOG.md

```diff
@@ -12,7 +12,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Hypothesis strategies for acquisition functions
 - `Kernel` base class allowing to specify kernels
 - `MaternKernel` class can be chosen for GP surrogates
-- `hypothesis` strategies and roundtrip test for kernels, constraints and objectives
+- `GammaPrior` class can be chosen for `MaternKernel`s
```

Comment:
Not quite accurate:
* It can be chosen for ALL kernels, not just the MaternKernel
* and it's not chosen as Kernel choice but for the lenghtscale of the kernel

Reply:
So what would you suggest? "`GammaPrior` can now be chosen as lengthscale prior"?

Reply:
Changed it accordingly, please un-resolve if un-happy